qoffer-domain/src/main/groovy/life/qbic/portal/portlet/offers/identifier/ProjectPart.groovy

```diff
@@ -0,0 +1,29 @@
+package life.qbic.portal.portlet.offers.identifier
+
+/**
+ * <class short description - 1 Line!>
+ *
+ * <More detailed description - When to use, what it solves, etc.>
+ *
+ * @since <versiontag>
+ */
```

Comment:
description missing

Reply:
```suggestion
/**
 * Represents the project part conserved of the identifier
 *
 * Contains the value of the project conserved part of the identifier
 *
 * @since 0.1.0
 */
```